src/AspNet.Security.OAuth.Alipay/AlipayAuthenticationHandler.cs

```diff
@@ -139,10 +139,11 @@ protected override async Task<OAuthTokenResponse> ExchangeCodeAsync([NotNull] OA
                 throw new HttpRequestException("An error occurred while retrieving user information.");
             }
 
-            using var payload = JsonDocument.Parse(await GetRawStringAsync(response, "alipay_user_info_share_response"));
+            var payload = await ReadJsonDocumentAsync(response, "alipay_user_info_share_response");
```

Comment:
```suggestion
            using var payload = await ReadJsonDocumentAsync(response, "alipay_user_info_share_response");
```

This one _does_ need a using.

Reply:
😂